source/help/apps/desktop-changelog.rst

```diff
@@ -1,6 +1,90 @@
 Desktop Application Changelog
 ========================================
 
+Release v4.7
+----------------------------
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+**Release day TBD**
+
+Breaking Changes or Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- 
+
+Highlights
+~~~~~~~~~~~~~~~
+- Added support for BrowserView.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+Windows
+^^^^^^^^^^^^^
+- Removed the dark mode toggle option for Windows.
+
+MacOS
+^^^^^^^^^^^^^
+-
+
+All Platforms
+^^^^^^^^^^^^^
+- Added Swedish language to the spellchecker.
+- Added a setting to specify the default desktop app download location.
+- Updated the desktop app loading indicator.
+- Restored deeplinking.
+- Switched the dictionary to the one Electron uses to have the ability to fix spelling.
```

Comment:
@amyblais lets please write these in terms of the user benefit, for example:

- Improved the spell check dictionary to provide more accurate spelling suggestions in more languages. The spell check language is now automatically based on the operating system setting.